lib/collection.go

```diff
@@ -198,3 +222,19 @@ func (collectionImpl) ListRawRefs(scope scope, p *EmptyParams) (string, error) {
 	}
 	return base.RawDatasetRefs(scope.Context(), scope.ActiveProfile().ID, scope.CollectionSet())
 }
+
+// Get gets the head of a dataset as a VersionInfo from the collection
+func (collectionImpl) Get(scope scope, p *CollectionGetParams) (*dsref.VersionInfo, error) {
```

Comment:
Name? 

`GetFromCollection`?
`Header`?
`DatasetDetails`?
`GetDatasetInfo`?

Reply:
I'm good with `Get`, as it is doing an analogous action to `datasets.Get`. The only other option I would go for might be `Head`, as long as the collection is only ever tracking head versions, and not used to provided historical information as well.

If hypothetically we landed on a struct name like `AggInfo` for "aggregated information" I could see changing this to a method name to match, but that's a discussion for later. In general, if a method name sounds like it is referring to a specific type of structure, it should return that same kind of structure, so that it reads like you're getting "the collection aggInfo for this InitID". Similar reasoning for why using `Head` might make sense.

Reply:
okay in keeping with speed/moving forward, going to keep this as `Get` and we can rename in a later pr when the feature is more settled.